microcosm_flask/conventions/crud.py

```diff
@@ -21,6 +21,10 @@
 
 class CRUDConvention(Convention):
 
+    @property
```

Comment:
I'm adding a hook here to choose a different pagination class. I'm hoping we don't actually need to use this hook.